src/components/app/Feature.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from '@rebass/grid';
+import Stack from 'stack-styled';
+import Heading from '../core/Heading';
+import Text from '../core/Text';
+
+const Feature = ({ as, imageComponent: Image, title, children }) => (
+	<Stack gap={2} as={as}>
```

Comment:
Same with `as`. This looks like a Pattern to me and thus should probably be opinionated about the markup it renders

Reply:
Then it will have obscure requirements, like if I'd hardcode `<li>` here, you'd have to remember to wrap it in `<ul>` or `<ol>`. Having no opinion on that makes this clear when you need it.

Reply:
I understand your intent but I think that should be encapsulated into a bigger component then and it should make this decision. It looks like an implementation detail to me ATM.

Reply:
> I don't think HTML semantics and ARIA are always implementation details

Hmm, I disagree with you on this one :) 

> And developers should be aware of HTML/ARIA all the time when they write markup

Should be doesn't mean they will be. In the ideal world, yes, they would do this but in reality oftentimes it's not possible to track this. Components solve this by encapsulating sane defaults.

Another benefit of such encapsulation is that it becomes consumable by another targets. Think react-native or Sketch renderers: they don't have HTML primitives nor ARIA attributes.

DS should server a source of truth for all possible consumers be it developers or designers. Also, different organizations might have different semantics when it comes to design language.

Reply:
Yeah, I disagree with this one ;-) In ideal world it might work, but in reality I'd rather work with HTML and ARIA instead of learning another API that tries to encapsulate all semantics of both.

Also developers will use components incorrectly the same way they use HTML/ARIA incorrectly. Code reviews and a11y testing may help with this problem, but just giving developers another API won't. And essentially you're punishing folks, who know HTML/ARIA, by forcing them to learn a new API, without solving the original issue of developers writing non-semantic markup.